src/test/java/teammates/ui/webapi/GetActionClassesActionTest.java

```diff
@@ -0,0 +1,41 @@
+package teammates.ui.webapi;
+
+import org.testng.annotations.Test;
+
+import teammates.common.util.Const;
+import teammates.ui.output.ActionClasses;
+
+/**
+ * SUT: {@link GetActionClassesAction}.
+ */
+public class GetActionClassesActionTest extends BaseActionTest<GetActionClassesAction> {
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.ACTION_CLASS;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return GET;
+    }
+
+    @Test
+    @Override
+    protected void testExecute() throws Exception {
+        ______TS("retrieve list of action class names");
+        GetActionClassesAction action = getAction();
+        action.execute();
+        JsonResult result = getJsonResult(action);
+        ActionClasses data = (ActionClasses) result.getOutput();
+
+        assertEquals(data.getActionClasses().size(), 95);
```

Comment:
This is unnecessary and is very prone to changes. The number `95` (or any other number here) has no significance.

Reply:
It is updated to checking whether the expected and the actual lists of action classes are the same.